ql/src/test/results/clientnegative/joinneg.q.out

```diff
@@ -1 +1 @@
-FAILED: SemanticException [Error 10004]: Line 6:12 Invalid table alias or column reference 'b': (possible column names are: x.key, x.value, y.key, y.value)
+FAILED: SemanticException [Error 10009]: Line 6:12 Invalid table alias 'b'
```

Comment:
The original error message was more informative.

Reply:
Agreed, but I think that's an issue for the new code.  I would rather remove the lines of code that converts to ExprNodes (but keep the unparsetranslator part.